scripts/bootstrap

```diff
@@ -28,9 +28,9 @@ while test $# -gt 0; do
     esac
 done
 if [ ${#flags[0]} -eq 0 ]; then
-    flags=("--dev" "--skip-lock")
+    flags=("--no-interaction" "--no-ansi")
```

Comment:
do we still need this complex options?

Reply:
`--no-interaction` is quite useful and probably necessary, `--no-ansi` makes logs more readable, but we can not use it if you prefer.